src/console/formatter.rs

```diff
@@ -0,0 +1,18 @@
+use std::fmt;
+
+pub struct Formatter {}
```

Comment:
The idiomatic thing to do when you don't need fields yet is to use a unit struct.

```rust
pub struct Formatter;
```